libs/scripts/tooltip.js

```diff
@@ -70,25 +70,21 @@ function setTooltipPosition(tooltips) {
   });
 }
 
-export default function addTooltipListeners(ownerElement) {
-  ownerElement?.addEventListener('click', () => {
-    ownerElement.classList.add('hide-tooltip');
-  });
+export default function addTooltipListeners() {
   ['keydown', 'mouseenter', 'focus', 'mouseleave', 'blur'].forEach((eventType) => {
     document.addEventListener(eventType, (event) => {
-      if (ownerElement && !ownerElement.classList.contains('hide-tooltip')
-        && eventType === 'keydown' && event.key === 'Escape') {
-        ownerElement.classList.add('hide-tooltip');
-      }
-
       const isTooltip = event.target?.matches?.('.milo-tooltip');
-      if (!isTooltip) return;
+      if (!isTooltip && eventType !== 'keydown') return;
 
       if (['mouseenter', 'focus'].includes(eventType)) {
         event.target.classList.remove('hide-tooltip');
         setTooltipPosition([event.target]);
       } else if (['mouseleave', 'blur'].includes(eventType)) {
         event.target.classList.add('hide-tooltip');
+      } else if (eventType === 'keydown' && event.key === 'Escape') {
+        const tooltip = document.querySelector('.milo-tooltip:not(.hide-tooltip)');
+        tooltip?.classList.add('hide-tooltip');
+        tooltip?.blur();
```

Comment:
To my knowledge, this is an anti-pattern. The focus doesn't move, it's on the same element, so a user could be confused as to the current focus position. Has this been vetted with someone on the accessibility side?

Reply:
We've discussed this further and latest decision is to remove the `blur()` call, as it can lead to confusion for users, whereas the use-case it's trying to solve is an edge case.